### ISSUE TITLE
Test for WCAG failure F14: Identifying content only by its shape or location

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#185]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F14
The objective of this technique is to show how identifying content only by its visual shape or location makes content difficult to understand and operate. When only visual identification or location is used, users with visual disabilities may find it difficult to locate content since they cannot see the screen or may perceive only a small portion of the screen at one time. Also, location of content can vary if page layout varies due to variations in font, window, or screen size.
## Example

The navigation instructions for a site state, "To go to next page, press the button to the right. To go back to previous page, press the button to the left." 
## Test procedure
1. Examine the Web page for textual references to content within the Web page.
2. Check that the references do not rely on only the visual shape or location of the content.
